webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js

```diff
@@ -106,7 +109,39 @@ const DetailsCard = ({
                 emptyState={<DefaultLoaderEmptyState />}
                 status={status}
               >
-                {hostgroupName}
+                {hostgroupId && (
+                  <div className="pf-u-display-flex pf-u-justify-content-space-between">
```

Comment:
There isn't a documentation in the pf4 website but this should also be using pf4 components
https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/layouts/Flex/examples/Flex.md